manifests/views.py

```diff
@@ -83,6 +85,12 @@ def list(self, request, *args, **kwargs):
 
         return res
 
+class SensorHardwareViewSet_CRUD(NodeAuthMixin, ModelViewSet):
+    queryset = SensorHardware.objects.all()
+    serializer_class = SensorHardwareCRUDSerializer
+    lookup_field = "hw_model"
+    authentication_classes = (NodeAuthMixin.authentication_classes[0],UserTokenAuthentication)
+    permission_classes = (NodeAuthMixin.permission_classes[0]|IsAdminUser,)    
```

Comment:
It looks like there's a little friction between merging the behavior from the NodeAuthMixin and other permission classes like IsAdminUser.

That's ok! We can still merge this work but let's make a note to revisit this kind of thing later so we can find a better approach.